src/Termonad/Config.hs

```diff
@@ -296,6 +296,23 @@ data ShowScrollbar
 
 -- }}}
 
+-- --< TabBar >-- {{{
+
+-- TODO: This data type has the same shape as ShowScrollBar.
+-- Perhaps we should instead have:
+--
+-- > data Sometimes = Never | Always | IfNeeded
+-- > newtype ShowScrollBar = ShowScrollBar Sometimes
+-- > newtype ShowTabBar = ShowTabBar Sometimes
```

Comment:
I think the code might be a little easier to document if we keep it like it is, instead of splitting it into `Sometimes` and newtypes.  I don't know if we really gain that much by factoring out `Sometimes`, other than a small amount of code reuse.  I don't think we will probably have any interesting combinators over `Sometimes`.

However, I don't feel super strongly about this either way.

Reply:
Feel free to make this decision and I will merge it either way.